### PR TITLE
DROOLS-3116 empty Literal Expression is preserved

### PR DESCRIPTION
**Pull request by tarilabs**
/cc @jomarko @manstis @etirelli requirement change as per DROOLS-3116

### CODE REVIEW

File: kie-wb-common-dmn/kie-wb-common-dmn-backend/src/test/java/org/kie/workbench/common/dmn/backend/DMNMarshallerTest.java [hunk 1]
```diff
@@ -1072,9 +1070,48 @@ public void test_wrong_context() throws IOException {
...
+        DMNMessage m0 = dmnModel.getMessages(DMNMessage.Severity.ERROR).get(0);
+        System.out.println(m0);
+        assertTrue("expected a message identifying the problem on a decision named 'my decision'",
+                   m0.getMessage().startsWith("No expression defined for name 'my decision'"));
+
+        DecisionNode d0 = dmnModel.getDecisionById("_cce32679-9395-444d-a4bf-96af8ee727a0");
+        // the identified DMN Decision is composed a literal expression missing text (text is null).
+        org.kie.dmn.model.api.Expression d0le = d0.getDecision().getExpression();
+        assertTrue(d0le instanceof org.kie.dmn.model.api.LiteralExpression);
+        assertEquals(null, ((org.kie.dmn.model.api.LiteralExpression) d0le).getText());
```
Comment (manstis) [inline kie-wb-common-dmn/kie-wb-common-dmn-backend/src/test/java/org/kie/workbench/common/dmn/backend/DMNMarshallerTest.java]: `assertNull` is probably better.

**Comment by tarilabs** [comment]
```
21:20:21 [INFO] jBPM :: Flow ....................................... FAILURE [  5.802 s]
```

Jenkins test again

**Comment by tarilabs** [comment]
```
Test Result (1 failure / ±0)
org.kie.workbench.common.services.backend.compiler.DefaultMavenCompilerTest.testJDTCompiler
```

Jenkins test again

**Comment by tarilabs** [comment]
```
Test Result (1 failure / ±0)
org.kie.workbench.common.services.backend.compiler.DefaultMavenCompilerTest.buildWithCloneTest
```

Jenkins test again

**Comment by tarilabs** [comment]
```
Test Result (1 failure / ±0)
org.kie.workbench.common.services.backend.compiler.DefaultMavenCompilerTest.buildWithCloneTest
```

Jenkins test again

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by tarilabs** [comment]
```
Test Result (1 failure / +1)
org.kie.workbench.common.services.backend.compiler.DefaultMavenCompilerTest.buildWithPullRebaseUberfireTest
```

Jenkins test again

**Comment by tarilabs** [comment]
Let's merge this while it's green LOL

**Pull request merged**